.github/workflows/pull.yml

```diff
@@ -156,15 +140,13 @@ jobs:
         uses: actions/upload-artifact@v4
         if: failure()
         with:
-          name: playwright-report
+          name: playwright-report-web
           path: packages/client/playwright-report/
           retention-days: 7
 
   openfin-end-to-end-test-fx:
     name: Openfin e2e test - FX
 
-    needs: web-end-to-end-test
```

Comment:
might as well kick them all off in parallel, no real dependency here

Reply:
vague chance that the OpenFin tests could interfere with Web ones as they are a bunch of dups, just running against OpenFin deployment (same BE ofc) .. it's working as it is, so I'd like to keep it like this in the interests of speed (takes a while to boot up OpenFin anyway) .. but if we get clashes, we can drop back to waiting for the Web E2E to finish, as before